database/20210806_000000_Add_Ticket_Hardcore.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `ticket` ADD `Hardcore` TINYINT(1) AFTER `ReportType`;
```

Comment:
I see `Hardcore` is later used as `mode` - I think calling it `Mode` here would be more consistent?